src/acbm/assigning/utils.py

```diff
@@ -287,6 +287,7 @@ def _get_activities_per_zone_df(activities_per_zone: dict) -> pd.DataFrame:
 def zones_to_time_matrix(
     zones: gpd.GeoDataFrame,
     id_col: Optional[str] = None,
+    time_units: str = "s",
```

Comment:
It might be worth not including a default value here. This would mean the units would always need to be specified  making the units that are expected visible in each calling context.

Reply:
Done in latest commit